rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-quickly.adoc

```diff
@@ -46,6 +46,5 @@ include::modules/rosa-sts-creating-a-cluster-quickly-cli.adoc[leveloffset=+1]
 * For information about the prerequisites to installing ROSA with STS, see xref:../rosa_planning/rosa-sts-aws-prereqs.adoc#rosa-sts-aws-prereqs[AWS prerequisites for ROSA with STS].
 * For details about using the `auto` and `manual` modes to create the required STS resources, see xref:../rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-with-customizations.adoc#rosa-understanding-deployment-modes_rosa-sts-creating-a-cluster-with-customizations[Understanding the auto and manual deployment modes].
 * For more information about using OpenID Connect (OIDC) identity providers in AWS IAM, see link:https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_providers_create_oidc.html[Creating OpenID Connect (OIDC) identity providers] in the AWS documentation.
-* For more information about troubleshooting ROSA cluster installations, see xref:../rosa_support/rosa-troubleshooting-installations.adoc#rosa-troubleshooting-installations[Troubleshooting installations].
-* For steps to contact Red Hat Support for assistance, see xref:../rosa_architecture/rosa-getting-support.adoc#rosa-getting-support[Getting support for Red Hat OpenShift Service on AWS].
-
+* For more information about troubleshooting ROSA cluster installations, see xref:../sd_support/rosa-troubleshooting-installations.adoc#rosa-troubleshooting-installations[Troubleshooting installations].
```

Comment:
Should [this link](https://github.com/openshift/openshift-docs/pull/56971/files#diff-70f029dfbd5101198efe7b42502bffe71bdfb40a41bc06981dafbaf5c6af04d9R54) be here also?

Reply:
@mburke5678 Looking at that link to the Troubleshooting installations, it seems fine and should be included.

Reply:
@EricPonvelle Sorry. I thought you missed one that is in the original PR. Not sure what I was looking at.